package.json

```diff
@@ -41,7 +36,82 @@
   "bugs": {
     "url": "https://github.com/absolute-version/commit-and-tag-version/issues"
   },
+  "eslintConfig": {
+    "extends": [
+      "prettier",
+      "eslint:recommended"
+    ],
+    "env": {
+      "node": true,
+      "es6": true,
+      "jest/globals": true
+    },
+    "overrides": [
+      {
+        "files": [
+          "test/**"
+        ],
+        "plugins": [
+          "jest"
+        ],
+        "extends": [
+          "prettier",
+          "eslint:recommended",
+          "plugin:jest/recommended"
+        ],
+        "rules": {
+          "jest/prefer-expect-assertions": "off",
+          "jest/expect-expect": "off"
+        }
+      }
+    ],
+    "parserOptions": {
+      "ecmaVersion": 2023
+    },
+    "plugins": [
+      "jest"
+    ],
+    "rules": {
+      "no-var": "error",
+      "no-unused-vars": [
+        "error",
+        {
+          "argsIgnorePattern": "_.*"
+        }
+      ]
+    }
+  },
   "homepage": "https://github.com/absolute-version/commit-and-tag-version#readme",
+  "jest": {
+    "cacheDirectory": "test-tmp",
+    "collectCoverage": true,
+    "collectCoverageFrom": [
+      "**/*.{js,jsx,ts}",
+      "!**/node_modules/**",
+      "!**/tmp/**",
+      "!**/test/**",
+      "!**/coverage/**",
+      "!**/bin/**"
+    ],
+    "coverageReporters": [
+      "lcov", "text"
+    ],
+    "projects": [
```

Comment:
Nice! I didn't know this was possible, and I like it.

Reply:
Yeah I was trying to figure out how to get things to run serially (for Fs safety) and also discovered I could configure some to run in parallel (default jest behaviour) and some serially